community/kernel/src/main/java/org/neo4j/kernel/impl/core/ThreadToStatementContextBridge.java

```diff
@@ -60,10 +60,16 @@ public void unbindTransactionFromCurrentThread()
     @Override
     public Statement instance()
     {
+        return tx().acquireStatement();
+    }
+
+    public KernelTransaction tx()
+    {
         checkIfShutdown();
-        return getKernelTransactionBoundToThisThread( true ).acquireStatement();
+        return getKernelTransactionBoundToThisThread( true );
```

Comment:
I don't like the `tx` method name. This method is part of the `get*TransactionBoundToThisThread` family of methods.